vm/devices/storage/disk_backend/src/lib.rs

```diff
@@ -16,6 +16,7 @@
 #![forbid(unsafe_code)]
 #![warn(missing_docs)]
 
+pub mod layered;
```

Comment:
given the scope of the implementation, I think it should get split out into its own crate, vs. hanging off the core disk_bakend.

Reply:
(especially now that it has its own resolve.rs, resolver.rs, etc...)

Reply:
Yeah OK